src/SFA.DAS.EAS.Web/Views/Shared/ValidationSummary.cshtml

```diff
@@ -2,7 +2,7 @@
 
 <div class='@(Html.ViewData.ModelState.IsValid ? "validation-summary-valid" : "validation-summary-errors") error-summary' role="group" data-valmsg-summary="true" >
     <h1 id="error-summary" class="heading-medium error-summary-heading">
-        There are errors on this page that need your attention
+        There is a problem
     </h1>
 
```

Comment:
The new design has increased the whitespace between the h1 and the error summary list. Has this been done?

Reply:
This is a common component and not part of this change.  It should be done in CON-403.

Reply:
I tried the new styles but they weren't applied.

Reply:
Chris has asked us to make the design changes individually for the pages we're changing, for now. The way I handled this in a previous PR, was to introduce a class based on the appropriate class from the new Gov Uk design system.

In this case, the relevant class is called `govuk-error-summary__title`, which should be applied to the `h2`. It needs a margin bottom of 15px, expanding to 20px at a breakpoint of 40.0625em. I can give you a hand with this if you want.

Reply:
Done. Thanks.